doc/man3/SSL_CTX_set1_sigalgs.pod

```diff
@@ -33,8 +33,11 @@ signature algorithms for B<ctx> or B<ssl>. The B<str> parameter
 must be a null terminated string consisting of a colon separated list of
 elements, where each element is either a combination of a public key
 algorithm and a digest separated by B<+>, or a TLS 1.3-style named
-SignatureScheme such as rsa_pss_pss_sha256. If a list entry is preceded
-with the C<?> character, it will be ignored if an implementation is missing.
+SignatureScheme such as rsa_pss_pss_sha256.
+The signature scheme and public key algorithm names (but not the digest names)
+are case-insensitive.
```

Comment:
This formulation is IMO more clear and points at the second interpretation above.

Reply:
The entire scheme name and only the pkey algorithm of the `+` form.